pkg/skaffold/deploy/helm.go

```diff
@@ -165,7 +180,62 @@ func (h *HelmDeployer) deployRelease(out io.Writer, r v1alpha2.HelmRelease, buil
 	if len(r.Overrides) != 0 {
 		os.Remove("skaffold-overrides.yaml")
 	}
-	return helmErr
+
+	return h.getDeployResults(ns, r.Name), helmErr
+}
+
+func (h *HelmDeployer) getReleaseInfo(release string) (*bufio.Reader, error) {
+	var releaseInfo bytes.Buffer
+	out := bufio.NewWriter(&releaseInfo)
+	if err := h.helm(out, "get", release); err != nil {
+		return nil, fmt.Errorf("error retrieving helm deployment info: %s", releaseInfo.String())
+	}
+	return bufio.NewReader(&releaseInfo), nil
+}
+
+func parseRuntimeObject(namespace string, b []byte) (Artifact, error) {
+	d := scheme.Codecs.UniversalDeserializer()
+	obj, _, err := d.Decode(b, nil, nil)
+	if err != nil {
+		return Artifact{}, fmt.Errorf("error decoding parsed helm deployment yaml: %s", err.Error())
+	}
+	return Artifact{
+		Obj:       &obj,
+		Namespace: namespace,
+	}, nil
+}
+
+func parseReleaseInfo(namespace string, b *bufio.Reader) []Artifact {
+	results := []Artifact{}
+	r := k8syaml.NewYAMLReader(b)
+	for {
+		doc, err := r.Read()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			logrus.Infof("error parsing helm deployment: %s", err.Error())
```

Comment:
should we have a `continue` here to avoid trying to parse an invalid or nil doc?

Reply:
good call, added